flake.nix

```
@@ -0,0 +1,13 @@
+{
+  inputs.flake-utils.url = github:numtide/flake-utils;
```

```suggestion
  inputs.flake-utils.url = "github:numtide/flake-utils";
```
my linter suggestions making this a string literal as url literals are apparently bad practice.